Firmware/first_lay_cal.cpp

```diff
@@ -96,65 +107,61 @@ bool lay1cal_load_filament(uint8_t filament)
+    static const char cmd_intro_mmu_4[] PROGMEM = "G1X5E29F1800";
+    static const char cmd_intro_mmu_5[] PROGMEM = "G1X55E8F2000";
+    static const char cmd_intro_mmu_6[] PROGMEM = "G1Z0.3F1000";
+    static const char cmd_intro_mmu_8[] PROGMEM = "G1X240E25F2200";
+    static const char cmd_intro_mmu_9[] PROGMEM = "G1Y-2F1000";
+    static const char cmd_intro_mmu_10[] PROGMEM = "G1X200E8F1400";
+    static const char cmd_intro_mmu_11[] PROGMEM = "G1Z0.2F1000";
+    static const char * const cmd_intro_mmu[] PROGMEM =
     {
         // first 2 items are only relevant if filament was not loaded - i.e. extraPurgeNeeded == true
         cmd_intro_mmu_3,
         cmd_intro_mmu_4,
-
         cmd_intro_mmu_5,
         cmd_intro_mmu_6,
         zero_extrusion,
         cmd_intro_mmu_8,
         cmd_intro_mmu_9,
         cmd_intro_mmu_10,
         cmd_intro_mmu_11,
-        cmd_intro_mmu_12,
     };
 
     if (MMU2::mmu2.Enabled())
     {
-        for (uint8_t i = (extraPurgeNeeded ? 0 : 2); i < (sizeof(intro_mmu_cmd)/sizeof(intro_mmu_cmd[0])); ++i)
+        for (uint8_t i = (extraPurgeNeeded ? 0 : 2); i < (sizeof(cmd_intro_mmu)/sizeof(cmd_intro_mmu[0])); ++i)
         {
-            enquecommand_P(static_cast<char*>(pgm_read_ptr(&intro_mmu_cmd[i])));
+            enquecommand_P(static_cast<char*>(pgm_read_ptr(&cmd_intro_mmu[i])));
         }
     }
     else
     {
-        static const char fmt1[] PROGMEM = "G1 X%d E%-.3f F1000";
-        enquecommandf_P(fmt1, 60, count_e(layer_height, extrusion_width * 4.f, 60));
-        enquecommandf_P(fmt1, 100, count_e(layer_height, extrusion_width * 8.f, 40));
+        enquecommandf_P(extrude_fmt_X, (float)60, count_e(layer_height, extrusion_width * 4.f, 60));
+        enquecommandf_P(extrude_fmt_X, (float)202.5, count_e(layer_height, extrusion_width * 8.f, 142.5));
     }
 }
 
 //! @brief Setup for printing meander
 void lay1cal_before_meander()
 {
-    static const char cmd_pre_meander_2[] PROGMEM = "G90"; //use absolute coordinates
-    static const char cmd_pre_meander_4[] PROGMEM = "G1 E-1.5 F2100";
-    static const char cmd_pre_meander_5[] PROGMEM = "G1 Z5 F7200";
-    static const char cmd_pre_meander_6[] PROGMEM = "M204 S1000"; //set acceleration
-    static const char cmd_pre_meander_7[] PROGMEM = "G1 F4000";
+#ifndef NEW_FIRST_LAYER_CAL
+    static const char cmd_pre_meander_4[] PROGMEM = "G1E-1.5F2100";
+    static const char cmd_pre_meander_5[] PROGMEM = "G1Z5F7200";
+#endif //NEW_FIRST_LAYER_CAL
+    static const char cmd_pre_meander_6[] PROGMEM = "M204S1000"; //set acceleration
 
     static const char * const cmd_pre_meander[] PROGMEM =
     {
             zero_extrusion,
-            cmd_pre_meander_2,
+            MSG_G90,
             MSG_M83, // use relative distances for extrusion
+#ifndef NEW_FIRST_LAYER_CAL
             cmd_pre_meander_4,
             cmd_pre_meander_5,
+#endif //NEW_FIRST_LAYER_CAL
             cmd_pre_meander_6,
-            cmd_pre_meander_7,
```

Comment:
The travel move will be at F7200, but I agree with this change. I always felt the travel is too slow for the first layer cal, too much oozing can happen.